webui_runner/Dockerfile

```diff
@@ -5,7 +5,12 @@ RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
 	apt-utils build-essential gpg-agent \
 	curl ca-certificates wget software-properties-common \
-	psmisc lsof git cmake nano
+	psmisc lsof git nano zlib1g-dev libedit-dev
+
+RUN wget -qO - https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add -
+RUN apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
+RUN apt-get update
+RUN apt-get install --yes cmake
```

Comment:
If this is to get a newer version of Cmake, why not update the base Docker image (ubuntu:bionic --> ubuntu:?); if not, maybe we should document why this requires a non-standard repo.

Reply:
Yes, bionic does not contain the version of cmake needed. I frankly forget which build component complained but I think it was WAMR. It's not uncommon to have to download a newer copy of cmake from a repository and that is the case here. Bionic has been the latest longterm release that most of these VMs are tested on and WASI SDK has an installer which was used, but 20.04 was released in June. I don't know what cmake comes default with 20.04 but if it is recent enough we can remove this here. I will add a comment.

Reply:
Note, I've attempted to upgrade from Bionic to Focal but it of course leads to new things breaking and more code changes. Notably wamr does not compile due to it not being able to find zlib even though it is installed:

>   Target "wamrc" links to target "ZLIB::ZLIB" but the target was not found.
>   Perhaps a find_package() call is missing for an IMPORTED target, or an
>   ALIAS target is missing?
> This warning is for project developers.  Use -Wno-dev to suppress it.

Not familiar with this error but it looks like it could be related to the cmake file included with wamr.